FreeOTP.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		2489072124910C89001C75A9 /* UIFont+Ext.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2489072024910C89001C75A9 /* UIFont+Ext.swift */; };
+		2489072324910D3B001C75A9 /* Screen.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2489072224910D3A001C75A9 /* Screen.swift */; };
```

Comment:
For reasons similar to above, can we rename `Screen.swift` to `Device.swift` which will be used to handle device specific changes/code like dynamic screen size.